manager/orchestrator/jobs/orchestrator.go

```diff
@@ -201,6 +201,11 @@ func (o *Orchestrator) handleEvent(ctx context.Context, event events.Event) {
 		service = ev.Service
 	case api.EventUpdateService:
 		service = ev.Service
+	case api.EventDeleteService:
+		if orchestrator.IsReplicatedJob(ev.Service) || orchestrator.IsGlobalJob(ev.Service) {
```

Comment:
Not a blocker, but wondering if we should have `IsJob()` and/or `IsService()` utility for these